docs/UserGuide.md

```diff
@@ -190,23 +296,25 @@ Expected outcome:
 
 ## Command summary
 
-| Action               | Format, Examples |
-|----------------------|------------------|
-| **Help**             | `help`           |
-| **List**             | `list`           |
-| **Clear**            | `clear`          |
-| **Exit**             | `exit`           |
-| **Add**              |                  |
-| **Delete**           |                  |
-| **Edit**             |                  |
-| **Status**           |                  |
-| **Remark**           |                  |
-| **Find by Company**  |                  |
-| **Find by Position** |                  |
-| **Filter**           |                  |
-| **Sort**             |                  |
-| **Stats**            |                  |
-| **Select**           |                  |
-| **Add Task**         |                  |
-| **Delete Task**      |                  |
+| Action               | Format, Examples              |
+|----------------------|-------------------------------|
+| **Help**             | `help`                        |
+| **List**             | `list`                        |
+| **Clear**            | `clear`                       |
+| **Exit**             | `exit`                        |
+| **Add**              |                               |
+| **Delete**           |                               |
+| **Edit**             |                               |
+| **Status**           | `status`, `status 1 o`        |
+| **Remark**           | `remark`, `remark 1 r/ hello` |
+| **Find by Company**  |                               |
+| **Find by Position** |                               |
+| **Filter**           |                               |
+| **Sort**             | `sort`, `sort a`. `sort b`    |
```

Comment:
```suggestion
| **Sort**             | `sort ORDER`<br> e.g., `sort a` |
```
Please follow this format. Also, I don't think `sort b` is a valid command?

Reply:
Thank you, I have taken note of it and changed it.